openstudiocore/src/gbxml/ReverseTranslator.cpp

```diff
@@ -725,26 +732,95 @@ namespace gbxml {
       }
 
       if (space && adjacentSpaceElements.size() == 2){
-
         QString adjacentSpaceId = adjacentSpaceElements.at(1).toElement().attribute("spaceIdRef");
         auto adjacentSpaceIt = m_idToObjectMap.find(adjacentSpaceId);
-        if (adjacentSpaceIt != m_idToObjectMap.end()){
+        if (adjacentSpaceIt != m_idToObjectMap.end()) {
           boost::optional<model::Space> adjacentSpace = adjacentSpaceIt->second.optionalCast<openstudio::model::Space>();
-          if (adjacentSpace){
-            // DLM: we have issues if interior ceilings/floors are mislabeled, override surface type for adjacent surfaces
-            // http://code.google.com/p/cbecc/issues/detail?id=471
+          if (adjacentSpace) {
+
             std::string currentSurfaceType = surface.surfaceType();
-            surface.assignDefaultSurfaceType();
-            if (currentSurfaceType != surface.surfaceType()){
-              LOG(Warn, "Changing surface type from '" << currentSurfaceType << "' to '" << surface.surfaceType() << "' for surface '" << surface.name().get() << "'");
+            if (currentSurfaceType == "RoofCeiling" || currentSurfaceType == "Floor") {
```

Comment:
@jasondegraw If the surfaceType attributes are set those should override the default surface type even if it is not a roof or a floor right?